plugins/processors/awsapplicationsignals/internal/resolver/kubernetes.go

```diff
@@ -525,36 +477,44 @@ func getKubernetesResolver(platformCode, clusterName string, logger *zap.Logger)
 
 		sharedInformerFactory := informers.NewSharedInformerFactory(clientset, 0)
 		podInformer := sharedInformerFactory.Core().V1().Pods().Informer()
+		err = podInformer.SetTransform(filterPodIPFields)
+		if err != nil {
+			logger.Error("failed to set custom transform into pod store", zap.Error(err))
```

Comment:
nit: change error message to something more meaningful
```suggestion
			logger.Error("failed to minimize pod object", zap.Error(err))
```

Reply:
Updated.